docs/source/contributing.rst

```diff
@@ -101,12 +101,12 @@ Start ``beamer-agent``::
 Submit a request::
 
     python scripts/call_contracts.py --deployment-dir deployments/ganache-local \
-                                     --keystore-file 0x1CEE82EEd89Bd5Be5bf2507a92a755dcF1D8e8dc.json \
+                                     --keystore-file 0xAC3812bF4f0FbD26323b2D9e74C494421550e2d9.json \
                                      --password '' \
                                      --eth-rpc http://localhost:8545 \
                                      request \
                                      --amount 1 \
-                                     --target-address 0x1CEE82EEd89Bd5Be5bf2507a92a755dcF1D8e8dc \
+                                     --target-address 0xAC3812bF4f0FbD26323b2D9e74C494421550e2d9 \
```

Comment:
This looks like needless key change -- note that `0x1CEE82EEd89Bd5Be5bf2507a92a755dcF1D8e8dc` is already known and compromised -- we should not be making more compromised keys, intentionally or by accident. :sweat_smile:

Reply:
after merging ape-ganache and release, we dont need to change any docs so will discard this at some point. and only waiting ape-ganache release as ape released 0.6.3